src/xrCore/xrDebug.cpp

```diff
@@ -712,7 +712,7 @@ void xrDebug::SetupExceptionHandler()
     else if (!strstr(commandLine, "-detailed_minidump"))
         minidumpFlags |= MiniDumpFilterMemory;
 #endif
-    
```

Comment:
Please, remove those styling changes

Reply:
Sorry, my editor automatically removes spaces at the end of lines.